instagram/src/components/PostContainer/PostHeader.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Posts.css';
```

Comment:
The file you have is `posts.css`.

For this import to work you have to change this line to import `./posts.css` or rename the CSS file to `Posts.css`.